code/datums/quirks/_quirk_static_data.dm

```diff
@@ -0,0 +1,65 @@
+GLOBAL_LIST_INIT_TYPED(all_quirk_static_data, /datum/quirk_static_data, generate_quirk_static_data())
+
+/// Constructs [GLOB.all_quirk_static_data] by iterating through a typecache of pregen data, ignoring abstract types, and instantiating the rest.
+/proc/generate_quirk_static_data()
```

Comment:
No! We have a preferences JSON asset. Do not send data like this over static data because it's enormous

Reply:
this isnt actually sent over static data, this is merely a bunch of singleton datum instances that are associated with [associated_key] in a global list

they exist to hold things initial() cant do, like the customization options (which is a list)

Reply:
i called it static data as im used to referring to singleton things like this as "static" (static lists, java static keywrod, w/e)